.github/workflows/paralleltest-docker.yml

```diff
@@ -13,14 +13,20 @@ on:
         description: adoptium/aqa-tests repository
         required: true
         type: string
-        default: 'adoptium/aqa-tests:v0.9.7-release'
+        default: 'adoptium/aqa-tests:v0.9.9-release'
       containerRegistry:
         description: Container registry
         required: true
       testLists:
         description: Number of testLists to generate
         required: false
-        default: 4
+        default: '4'
+      pool:
+          description: Agent pool
+          required: true
+          type: string
+          default: None
+        default: '4'
 
 jobs:
   configuration:
```

Comment:
`ubuntu-latest` delivers an agent faster. We could use `${ inputs.pool }` to be more consistent and, potentially, more secure.

Reply:
Because we are asked to use Microsoft 1ES / deemed secure agents in everything we do, I would say keep it to `${ inputs.pool }`. 

We can always provide the 1ES team with more hit-counts on any bugs they are monitoring about provisioning time, and it is our responsibility to do this kind of thing rather than working around the intended system anyways :).